website/docs/usage/training.mdx

```diff
@@ -575,6 +575,29 @@ now-updated model to the predicted docs.
 
 </Infobox>
 
+### Using rehearsing to address catastrophic forgetting {id="rehearse-components", tag="experimental", version="3.5.1"}
+
+Perform “rehearsal” updates to pre-trained components. Rehearsal updates teach the current component to make predictions similar to an initial model, to try to address the “catastrophic forgetting” problem. This feature is experimental.
+
+```ini {title="config.cfg (excerpt)"}
+[nlp]
+pipeline = ["sentencizer", "ner", "entity_linker"]
+
+[components.ner]
+source = "en_core_web_sm"
+
+[training]
+rehearse_components = ["ner"]
+```
+
+<Infobox variant="warning" title="Loss calculation" id="rehearse-components-loss">
+
+Be aware that the loss is calculated by the sum of both the `update` and `rehearse` function. 
```

Comment:
I'm not sure I follow the "sum of both `update` and `rehearse`" phrase?

Reply:
I agree; This might be a bit unclear. 
What I meant was: "The loss is the sum of both the `update` loss and the `rehearse` loss."